packages/framework/fluid-static/src/fluidContainer.ts

```diff
@@ -145,12 +156,12 @@ export interface IFluidContainer extends IEventProvider<IFluidContainerEvents> {
 }
 
 /**
- * Implementation of the IFluidContainer.
+ * {@inheritDoc IFluidContainer}
  */
 export class FluidContainer extends TypedEventEmitter<IFluidContainerEvents> implements IFluidContainer {
     private readonly connectedHandler = () => this.emit("connected");
     private readonly disconnectedHandler = () => this.emit("disconnected");
-    private readonly disposedHandler = () => this.emit("disposed");
+    private readonly disposedHandler = () => this.emit("dispose");
```

Comment:
This is a breaking change. What's the thought behind this change?

Reply:
Ah, not intentional. I had initially experimented with some type-aliasing to enhance the docs presentation, but abandoned that. Will restore :)